.github/workflows/build_win.yml

```diff
@@ -3,32 +3,74 @@ name: Build nRF Connect for Desktop for Windows x64
 on:
     workflow_dispatch:
         inputs:
+            sign_windows:
+                type: boolean
+                default: false
+                required: true
     workflow_call:
 
 jobs:
     build:
         runs-on: windows-latest
         steps:
             - uses: actions/checkout@v4
-            - name: Build
-              uses: ./.github/actions/build-action
 
             - name: Fetch certificate file
               shell: bash
               env:
-                  NORDIC_SIGNING_CERTIFICATE:
-                      ${{ secrets.NORDIC_SIGNING_CERTIFICATE_PFX }}
-              run:
-                  echo "$NORDIC_SIGNING_CERTIFICATE" | base64 --decode >
-                  ./nordic_signing_certificate.pfx
-              if: github.event_name != 'pull_request'
+                  SM_CLIENT_CERT_FILE_B64:
+                      ${{ secrets.NORDIC_SM_CLIENT_CERT_P12 }}
+              run: |
+                  echo "$SM_CLIENT_CERT_FILE_B64" | base64 --decode > /d/Certificate_pkcs12.p12
+                  cat /d/Certificate_pkcs12.p12
```

Comment:
This would print the certificate in the log, right? 😱

Reply:
Fixed 😱